pandas/_typing.py

```diff
@@ -211,7 +211,6 @@
 # TODO: add Ellipsis, see
 # https://github.com/python/typing/issues/684#issuecomment-548203158
 # https://bugs.python.org/issue41810
-PositionalIndexer = Union[int, np.integer, slice, Sequence[int], np.ndarray]
-PositionalIndexer2D = Union[
-    PositionalIndexer, Tuple[PositionalIndexer, PositionalIndexer]
-]
+PositionalIndexer = Union[int, np.integer, slice, List[int], np.ndarray]
```

Comment:
can you add a comment as to why this is List[int] instead of Sequence[int]?  I almost asked about it before reading back through the thread

Reply:
done in next commit